x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/__test__/MetadataTable.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import 'jest-dom/extend-expect';
+import { render, cleanup } from 'react-testing-library';
+import { MetadataTable } from '..';
+import {
+  expectTextsInDocument,
+  expectTextsNotInDocument
+} from '../../../../utils/testHelpers';
+import { Transaction } from '../../../../../typings/es_schemas/ui/Transaction';
+
+describe('MetadataTable', () => {
+  afterEach(cleanup);
+  describe('required sections', () => {
+    it('shows message if no data is available', () => {
```

Comment:
Maybe a little clearer: 
```suggestion
    it('shows "empty state message" if no data is available', () => {
```